svthalia/Reaxit
# Fix empty albums

### Summary
This fixes a crash that happens when the album cover is null

### How to test
Steps to test the changes you made:
1. Go to albums on staging
2. See the placeholder cover
